newsfragments/README.md

```diff
@@ -5,15 +5,17 @@ release notes. This should be a description of aspects of the change
 commit message and PR description, which are a description of the change as
 relevant to people working on the code itself.)
 
- Each file should be named like `<ISSUE>.<TYPE>.rst`, where
+Each file should be named like `<ISSUE>.<TYPE>.rst`, where
 `<ISSUE>` is an issue number, and `<TYPE>` is one of:
 
 * `feature`
 * `bugfix`
+* `performance`
 * `doc`
+* `internal`
+* `removal`
 * `misc`
-* `breaking-change`
```

Comment:
@kclowes any reason we'd want to keep these different for `web3.py`? I needed the `internal` and so I just copy / pasted from the template project. Makes things easier.

Reply:
Nope, just hadn't gotten around to changing it. Thanks!